From 56f7d9aa223d9123174f6eec9683111f063794fc Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <ignatenkobrain@fedoraproject.org>
Date: Wed, 1 Nov 2017 08:07:10 +0100
Subject: [PATCH] set SOVERSION for bpf-shared

Signed-off-by: Igor Gnatenko <ignatenkobrain@fedoraproject.org>
---
 src/cc/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/cc/CMakeLists.txt b/src/cc/CMakeLists.txt
index c31936e..b552f45 100644
--- a/src/cc/CMakeLists.txt
+++ b/src/cc/CMakeLists.txt
@@ -25,6 +25,7 @@ include(static_libstdc++)
 add_library(bpf-static STATIC libbpf.c perf_reader.c)
 set_target_properties(bpf-static PROPERTIES OUTPUT_NAME bpf)
 add_library(bpf-shared SHARED libbpf.c perf_reader.c)
+set_target_properties(bpf-shared PROPERTIES VERSION ${REVISION_LAST} SOVERSION 0)
 set_target_properties(bpf-shared PROPERTIES OUTPUT_NAME bpf)
 
 set(bcc_common_sources bpf_common.cc bpf_module.cc exported_files.cc)
-- 
2.14.3

